app/javascript/components/bootcamp/SolveExercisePage/exercises/Exercise.ts

```diff
@@ -63,6 +63,23 @@ export abstract class Exercise {
     this.animations.push(animation)
   }
 
+  public didFunctionOccurNTimes(
+    result: InterpretResult,
+    fnName: string,
+    times: number
+  ): boolean {
+    return result.getters.getFunctionOccurenceInCode(fnName) === times
+  }
```

Comment:
I'd rather this was: `numTimesFunctionOccurred` and then use the matchers in the check for the comparison. So we can then have 1 time or <2, or >5 etc